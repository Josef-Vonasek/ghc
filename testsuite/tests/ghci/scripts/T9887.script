:l T9887.hs
