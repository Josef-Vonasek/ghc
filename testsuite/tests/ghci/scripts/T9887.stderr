
T9887.hs:6:21:
    The static declaration is not supported when producing byte-code. Please use -fobject-code.
    In the first argument of ‘deRefStaticPtr’, namely ‘(static True)’
    In the expression: deRefStaticPtr (static True)
    In an equation for ‘f’: f = deRefStaticPtr (static True)
